.circleci/config.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
+jobs:
+  build:
+    machine: true
+    working_directory: ~/eventuate-messaging-kafka-quarkus
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - eventuate-messaging-kafka-quarkus-{{ checksum "gradle.properties" }}
```

Comment:
Please add checksum of wrapper properties

Reply:
ok